packages/expo-av/android/build.gradle

```
@@ -79,4 +79,6 @@ dependencies {
 
   api "com.squareup.okhttp3:okhttp:3.10.0"
   api "com.squareup.okhttp3:okhttp-urlconnection:3.10.0"
+
+  api 'com.android.support:support-annotations'
```

```suggestion
  api "com.android.support:support-annotations:${safeExtGet("supportLibVersion", "28.0.0")}"
```

Let's use what we use everywhere else and conform to `supportLibVersion` setting

https://github.com/expo/expo/blob/eeecf97a3a36855daee8d6393508bedcd1c423cc/packages/expo-image-manipulator/android/build.gradle#L73